qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -134,15 +135,25 @@ def set_cmd_text_command(self, text: str,
             count: The count if given.
             space: If given, a space is added to the end.
             append: If given, the text is appended to the current text.
+            insert: If given, the text is inserted in the current text
+                    at cursor position. This cannot be used with append
             run_on_count: If given with a count, the command is run with the
                           given count rather than setting the command text.
         """
         if space:
             text += ' '
+        if append and insert:
```

Comment:
Use `cmdutils.check_exclusive` instead (see e.g. `qutebrowser/browser/commands.py` for how it is used)